cli/installer/kubernetes.go

```diff
@@ -21,7 +25,258 @@ var kubernetes = installer{
+				",ingress.hosts[0].paths[0].path=/,ingress.hosts[0].paths[0].pathType=Prefix'",
+		)
+	}
+
+	execCmd(helmCmd(conf, cmd...), ui)
+
+	psql := "host=tracetest-postgresql user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+	c := getTracetestConfigFileContents(psql, ui, conf)
+	ttc := createTmpFile("tracetest-config", string(c), ui)
+	defer os.Remove(ttc.Name())
+	execCmd(
+		kubectlNamespaceCmd(conf,
+			"create configmap tracetest --from-file="+ttc.Name()+" -o yaml --dry-run=client",
+			"| sed 's#"+path.Base(ttc.Name())+"#config.yaml#' |",
+			kubectlNamespaceCmd(conf, "replace -f -"),
+		),
+		ui,
+	)
+
+	cc := createTmpFile("collector-config", string(getCollectorConfigFileContents(ui, conf)), ui)
+	defer os.Remove(cc.Name())
+
+	execCmd(
+		kubectlNamespaceCmd(conf,
+			"create configmap collector-config --from-file="+cc.Name()+" -o yaml --dry-run=client",
+			"| sed 's#"+path.Base(cc.Name())+"#collector.yaml#' |",
+			kubectlNamespaceCmd(conf, "replace -f -"),
+		),
+		ui,
+	)
+
+	execCmd(kubectlNamespaceCmd(conf, "delete pods -l app.kubernetes.io/name=tracetest"), ui)
+	execCmd(kubectlNamespaceCmd(conf, "delete pods -l app.kubernetes.io/name=otel-collector"), ui)
+
+	if conf.Bool("demo.enable") {
+		execCmd(fmt.Sprintf(demoScript, conf.String("k8s.namespace")), ui)
+	}
+
+	ui.Success("Install successful!")
+	ui.Println(fmt.Sprintf(`
+To access tracetest:
+
+	%s
+
+Then, use your browser to navigate to:
+
+  http://localhost:8080
+
+Happy TraceTesting =)
+`, kubectlNamespaceCmd(conf, "port-forward svc/tracetest 8080")))
+
+}
+
+func helmCmd(config configuration, cmd ...string) string {
+	return fmt.Sprintf(
+		"helm --kubeconfig %s --kube-context %s %s",
+		config.String("k8s.kubeconfig"),
+		config.String("k8s.context"),
+		strings.Join(cmd, " "),
+	)
+}
+
+const (
+	certManagerYaml          = "https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml"
+	certManagerClusterIssuer = `
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: selfsigned
+spec:
+  selfSigned: {}
+`
+	jaegerOperatorYaml = "https://github.com/jaegertracing/jaeger-operator/releases/download/v1.32.0/jaeger-operator.yaml"
+	jaegerManifest     = `
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+`
+	collectorYaml = "https://raw.githubusercontent.com/kubeshop/tracetest/main/k8s/collector.yml"
+
+	demoScript = `
+tmpdir=$(mktemp -d)
+curl -L https://github.com/kubeshop/pokeshop/tarball/master | tar -xz --strip-components 1 -C  $tmpdir
+cd $tmpdir/helm-chart
+helm dependency update
+
+helm upgrade --install demo . \
+  --namespace demo --create-namespace \
+  -f values.yaml \
+  --set image.tag=latest \
+  --set image.pullPolicy=Always \
+  --set postgres.auth.username=ashketchum,postgres.auth.password=squirtle123,postgres.auth.database=pokeshop \
+  --set rabbitmq.auth.username=guest,rabbitmq.auth.password=guest,rabbitmq.auth.erlangCookie=secretcookie \
```

Comment:
I think you forgot to set the jaeger config. I cannot get traces back.

Reply:
good catch. It's fixed now

Reply:
![Screenshot from 2022-09-13 14-02-31](https://user-images.githubusercontent.com/2704737/189962879-d5d15f9f-ed4f-4b27-b29e-abbaef70b903.png)
The Host is correct, but the port is set as the `namespace` instead.

Reply:
Fixed!